_maps/map_files/LV624/LV624.dmm

```diff
@@ -10200,8 +10200,12 @@
 /turf/open/ground/river,
 /area/lv624/ground/jungle6)
 "aNa" = (
-/obj/machinery/door/poddoor/timed_late/containment/landing_zone/lz2{
-	dir = 1
+/obj/machinery/door/airlock/mainship/engineering{
+	density = 0;
+	icon_state = "door_open";
+	name = "\improper Robotics Dome";
+	req_access_txt = "0";
+	req_one_access_txt = "0"
```

Comment:
These var-edits are not ideal.
Make a child of the airlock with different `density`.
Also preefer `req_access` to `req_access_txt`, same for the other one.